src/utils/base-card.ts

```diff
@@ -49,6 +50,23 @@ export class MushroomBaseCard extends MushroomBaseElement {
             : null;
     }
 
+    protected renderBatteryBadge(entity: HassEntity): TemplateResult | null {
+        const level = entity.attributes.battery_level;
+        const color = level > 40 ? "var(--rgb-success)" : level > 20 ? "var(--rgb-warning)" : "var(--rgb-danger)";
```

Comment:
```suggestion
        const color = level >= 70 ? "var(--rgb-success)" : level >= 30 ? "var(--rgb-warning)" : "var(--rgb-danger)";
```
Home Assistant battery color rules for tile card. May be we can use the same values ?